keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -203,23 +206,22 @@ open class SqlResourceRepository(
   override fun eventHistory(id: String, limit: Int): List<ResourceHistoryEvent> {
     require(limit > 0) { "limit must be a positive integer" }
 
-    val resource = get(id)
-
     return sqlRetry.withRetry(READ) {
       jooq
         .select(EVENT.JSON)
         .from(EVENT)
         // look for resource events that match the resource...
         .where(
           EVENT.SCOPE.eq(EventScope.RESOURCE)
-            .and(EVENT.REF.eq(resource.uid))
+            .and(EVENT.REF.eq(id))
```

Comment:
I no longer have to do an `in` with a sub-select here which was terrible for performance